docs/reference/crds/verifiers.md

```diff
@@ -9,6 +9,7 @@ spec:
   name: required, name of the verifier
   artifactType: required, the type of artifact this verifier handles
   address: optional. Plugin path, defaults to value of env "RATIFY_CONFIG" or "~/.ratify/plugins"
+  version: optional. Version of the plugin implementation, defaults to 1.0.0
```

Comment:
Is there any other values that users can choose to fill it out in this field apart from 1.0.0?

Reply:
Yes, the verion depends on the range of version the verifier supports. For example, the SBOM verifier is at 2.0.0 alpha

Reply:
Adding an index table to list all available versions of external plugins would be much more friendly. Pointing to plugin code is not that friendly.

Reply:
I feel the supported versions of each verifier should be listed in the verifier repo and Ratify-web has link to it. However, currently we don't have it in the Ratify repo yet.

Reply:
As discussed during community, i ve added the index page to include info from plugin code. Please review when convenient thanks!